R/functions.R

```diff
@@ -103,8 +123,18 @@ clique_test = function(Y,Z,Z_a,Z_b,Zobs_id,minr,minc,...){
   rtest_out = list(tobs=tobs,tvals=tvals)
   decision = out_pval(rtest_out,ret_pval,alpha=0.05)
 
+  valid_clique <- apply(conditional_clique, 2, function(x) length(unique(x))==1)
+  if (sum(valid_clique)!=0){
+    cat("The biclique containing Zobs is not valid \n")

```

Comment:
Could you change the error here from "valid" to something a bit more descriptive, like "the biclique containing Zobs contains only one exposure and will therefore result is a powerless test".  Just don't want to confuse valid with statistical validity.

Reply:
I've changed "valid" to "only one exposure for some focal assignment, and therefore the result is a powerless test", and change the corresponding return of the function.